api/all.yaml

```diff
@@ -2289,6 +2308,23 @@ components:
       required:
         - principal
         - privileges
+    GetMetastoreSummaryResponse:
+      type: object
+      properties:
+        metastore_id:
+          description: Unique identifier for the metastore.
```

Comment:
Is this one guaranteed to be a UUID or can it be anything?

Reply:
The open api spec accepts any unique identifier string. The server implementation uses UUID as the unique identifier

Reply:
We can add `format: uuid` in that case to make it clear.

https://stackoverflow.com/questions/50204588/how-to-define-uuid-property-in-json-schema-and-open-api-oas

Reply:
I don't know if that's a good idea as a different implementation of this spec could use any other unique identifier?